src/_userflair.scss

```
@@ -4,7 +4,7 @@
 	display: inline-flex;
 	align-items: center;
 	vertical-align: middle;
-	padding: 0 0 0 16px;
+	padding: 0 0 0 0px;
```

Minor nitpick: This can be simplified since we're not trying to set a different value for certain sides anymore

```suggestion
	padding: 0;
```